doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-scenario-template-con.adoc

```diff
@@ -0,0 +1,6 @@
+[id='test-designer-create-test-scenario-template-con']
+= Test scenario template
+
+Before specifying test scenario definitions, you need to create a test scenario template. The header of the test scenario table defines the template for each scenario. You need to set the types of the instance and property headers for both the GIVEN and EXPECTED sections. Instance headers map to a particular data object (a fact), whereas the property headers map to a particular field of the corresponding data object.
```

Comment:
It is `EXPECT` not `EXPECTED`. Please update the entire doc

Reply:
Corrected the naming of the UI element to *EXPECT* as suggested.